client/my-sites/stats/subscribers-section/subscribers-navigation-arrows.tsx

```diff
@@ -0,0 +1,77 @@
+import NavigationArrows from '../navigation-arrows';
+
+type SubscribersNavigationProps = {
+	date: Date;
+	period: string;
+	quantity: number;
+	onDateChange: ( newDate: Date ) => void;
+};
+
+const SubscribersNavigation = ( {
+	date,
+	period,
+	quantity,
+	onDateChange,
+}: SubscribersNavigationProps ) => {
+	const isFuture = ( date: Date ): boolean => {
+		const incomingNextDate = new Date( date );
+		incomingNextDate.setDate( incomingNextDate.getDate() + 1 );
+		// const incomingNextDate = new Date( date + 24 * 60 * 60 * 1000 );
+		incomingNextDate.setHours( 0, 0, 0, 0 );
+
+		const today = new Date();
+		today.setHours( 0, 0, 0, 0 );
+
+		return incomingNextDate.getTime() >= today.getTime();
+	};
+
+	const calculateDate = ( add: boolean ): Date => {
+		switch ( period ) {
```

Comment:
It reminds me of this:
https://gist.github.com/jonasraoni/a7bd7c5ca328543d88afa94e9ef2d184

```JavaScript
const signedQuantity = (-1) ** add * quantity;
```

The tricky part is not a suggestion 😄 But I do prefer a `signedQuantity` calculated at the beginning of the function, which would simply the function a bit.